pkg/kubelet/kubelet.go

```diff
@@ -2289,32 +2290,74 @@ func (kl *Kubelet) rejectPod(pod *api.Pod, reason, message string) {
 		Message: "Pod " + message})
 }
 
+// getNodeAnyWay() must return a *api.Node which is required by RunGeneralPredicates().
+// The *api.Node is obtained as follows:
+// 1. When kubelet is in standaloneMode, manufactures *api.Node with kubelet
```

Comment:
Since we cannot get *api.Node (needed by RenGeneralPredicates()) from apiserver in standaloneMode, we must somehow "make" the *api.Node out of the Kubelet's local information.